fuzzers/libfuzzer_libmozjpeg/src/lib.rs

```diff
@@ -128,7 +129,8 @@ fn fuzz(corpus_dirs: &[PathBuf], objective_dir: PathBuf, broker_port: u16) -> Re
     let mut fuzzer = StdFuzzer::new(scheduler, feedback, objective);
 
     // The wrapped harness function, calling out to the LLVM-style harness
-    let mut harness = |buf: &[u8]| {
+    let mut harness = |input: &BytesInput| {
```

Comment:
That could be `I: HasTargetBytes`, that way we support more Inputs automatically